BREAKING-CHANGES.md

```diff
@@ -44,6 +45,17 @@ This is a quick summary of the sections below:
 
 ## v0.26.0 (unreleased)
 
+### Remove deprecated `Block::title_on_bottom` ([#757])
+
+[#757]: https://github.com/ratatui-org/ratatui/pull/757
+
+`Block::title_on_bottom` was deprecated in v0.22. Use `Block::title` and `Title::position` instead.
+
+```diff
+- block.title("foobar").title_on_bottom();
++ block.title(Title::from("foobar").position(Position::Bottom));
+```
```

Comment:
If we come up with a new API for title in 0.26, we may reference `bottom_title` here.